private/buf/cmd/buf/command/mod/modinit/modinit.go

```diff
@@ -43,9 +43,9 @@ func NewCommand(
 ) *appcmd.Command {
 	flags := newFlags()
 	return &appcmd.Command{
-		Use:   name,
+		Use:   name + " <input>",
```

Comment:
I'm not sure this counts as an "input" you will need to have a look at the precedent for `input` elsewhere in the codebase. This may be better off as `argument` or `arg`... I'm not sure, see what you can find

Reply:
I checked the precedence of ```input``` in the codebase. Looks like ```input``` refers to files in other commands. I didn't find ```argument``` or ```arg``` elsewhere. I think ```module-name``` will be a better way to describe.

Reply:
[module-name](https://docs.buf.build/tour/push-a-module#module-names) looks appropriate.